nixos/modules/services/x11/display-managers/sx.nix

```diff
@@ -0,0 +1,39 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.services.xserver.displayManager.sx;
+
+in {
+  options = {
+    services.xserver.displayManager.sx = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Whether to enable the "sx" pseudo-display manager, which allows users
```

Comment:
Maybe `mkEnableOption`?

Reply:
`mkEnableOption` specifies a short description, not sure if thats the right thing to do here

Reply:
You could do something like:
```
enable = (mkEnableOption "'sx' pseudo-display manager") // { description = ''
...
''})
```